charts/fluid/fluid/crds/data.fluid.io_databackups.yaml

```diff
@@ -1,13 +1,32 @@
 
 ---
-apiVersion: apiextensions.k8s.io/v1
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
You need to upgrade controller-gen version. Currently Fluid is on CRD v1.

Reply:
Thx. Fixed.